This is to avoid running demos on `make install`, which break cross compile

--- a/Makefile
+++ b/Makefile
@@ -294,7 +294,7 @@ endif
 # just compile GraphBLAS
 gb:
 	echo $(CMAKE_OPTIONS)
-	( cd GraphBLAS && $(MAKE) JOBS=$(JOBS) CMAKE_OPTIONS='$(CMAKE_OPTIONS)' )
+	( cd GraphBLAS && $(MAKE) JOBS=$(JOBS) CMAKE_OPTIONS='$(CMAKE_OPTIONS)' library )
 
 # compile and install GraphBLAS libgraphblas_renamed, for MATLAB
 gbrenamed:
@@ -309,7 +309,7 @@ gbrenamed:
 
 # just compile Mongoose
 mon:
-	( cd Mongoose && $(MAKE) CMAKE_OPTIONS='$(CMAKE_OPTIONS)' )
+	( cd Mongoose && $(MAKE) CMAKE_OPTIONS='$(CMAKE_OPTIONS)' library )
 
 # compile and install Mongoose
 moninstall: mon
